### PR TITLE
break: BREAKING CHANGE update default transport

### PR DESCRIPTION
**Pull request by lennoxdavidjr**
* BREAKING CHANGE

### CODE REVIEW

**Comment by ryanhallcs** [comment]
LGTM - can you update the readme as well?

File: README.md [hunk 1]
```diff
@@ -1,9 +1,9 @@
 # event-sourcerer-client
 Event Sourcing client
 
-This client would be in charge of handling publishing of events to the event sourcing service.
+This client would be in charge of handling publishing of events to other services.
 
-The intent is for this client to be an abstraction over whatever protocol/transport/service we decide to use for event publishing and consumption.
+The intent is for this client to be the vehicle upon which our service .
```
Comment (ryanhallcs) [inline README.md:6]:
```suggestion
The intent is for this client to be the vehicle upon which our service broadcasts event-sourced data.
```

File: README.md [hunk 2]
```diff
@@ -12,22 +12,10 @@ The intent is for this client to be an abstraction over whatever protocol/transp
 // Default configuration
 const Jackrabbit = require('@pager/jackrabbit');
 const { Client } = require('@pager/event-sourcerer-client');
-const exchange = Jackrabbit('amqp://localhost').topic('sourcerer');
+const exchange = Jackrabbit('amqp://localhost').topic('events');
 const client = Client.default(exchange);
 ```
 
-```javascript
-// Manual setup
```
Comment (ryanhallcs) [inline README.md:20]: we should still provide a manual setup example, and replace `SingleKeyRabbitTransport` with `EventTransport`

File: README.md [hunk 3]
```diff
@@ -64,10 +43,23 @@ After using this function, all publish payloads with this stream + eventType wil
...
 
+#### EventsTransport
+
+This transport is the primary method used to publish messages to a RabbitMQ instance, under a single key by stream. The key being ``.
+
 #### SingleKeyRabbitTransport
 
 This transport can be used to publish messages to a RabbitMQ instance, under a single key by stream. The key being `events.{stream}.created`.
 
+##### `addPrePublishValidator(stream, eventType, schema)`
```
Comment (ryanhallcs) [inline README.md:54]: why was this moved to be a subsection of `SingleKeyRabbitTransport`?

**Comment by ryanhallcs** [comment]
can you rename this pr `break: ` instead of `feat: ` to trigger a semver major bump?

**Comment by lennoxdavidjr** [comment]
Ugh. Looks like my changes were not pushed up. This was a draft @ryanhallcs  Updating.

File: README.md (hunk 3 above)
Comment (lennoxdavidjr) [reply README.md:54]: Replaced!

File: README.md (hunk 2 above)
Comment (lennoxdavidjr) [reply README.md:20]: Added in the example with updated values

File: README.md (hunk 1 above)
Comment (lennoxdavidjr) [reply README.md:6]: :facepalm: Updated!
